File: WhichPharmaPortal/WhichPharma.Models/Client/RFQSummary.cs
using System;
using WhichPharma.Models.Attributes;

namespace WhichPharmaPortal.Models.Client
{
    public class RFQSummary
    {
        public string Number { get; set; }
        public DateTime CreationDate { get; set; }
        public string Title { get; set; }
        public RFQState State { get; set; }
        public DateTime StateChangeDate { get; set; }
        [Optional]
        public string AssigneeUsername { get; set; }
        [Optional]
        public string Reason { get; set; }
        [Optional]
        public long? Reminder { get; set; }
        [Optional]
        public DateTime? EndingDate { get; set; }
    }
}

File: WhichPharmaPortal/WhichPharmaPortal/ClientApp/src/models/RFQSummary.ts
import { RFQState } from './RFQState';

export interface RFQSummary {
    number: string;
    creationDate: string;
    title: string;
    state: RFQState;
    endingDate: string;
    stateChangeDate: string;
    assigneeUsername?: string;
    reason?: string;
    reminder?: number;
}
